packages/suite/src/services/google.ts

```diff
@@ -208,14 +217,28 @@ class Client {
                 });
 
                 const json = await res.json();
-                Client.accessToken = json.access_token;
-                Client.refreshToken = json.refresh_token;
+                if (!json?.access_token || !json?.refreshToken) {
+                    throw new Error('Could not retrieve the tokens.');
+                } else {
+                    Client.accessToken = json.access_token;
+                    Client.refreshToken = json.refresh_token;
+                }
```

Comment:
```suggestion
                Client.accessToken = json.access_token;
                Client.refreshToken = json.refresh_token;
```

Reply:
Done in 469efbf93ea70aef646477c7b77bf76e1481be54